app/auth/actions.ts

```diff
@@ -0,0 +1,84 @@
+"use server";
+
+import { revalidatePath } from "next/cache";
+import { redirect } from "next/navigation";
+
+import { createClient } from "@/utils/supabase/server";
+
+export async function login(formData: FormData) {
+  const supabase = createClient();
+
+  // type-casting here for convenience
+  // in practice, you should validate your inputs
+  const data = {
+    email: formData.get("email") as string,
+    password: formData.get("password") as string,
+  };
+
+  const { error } = await supabase.auth.signInWithPassword(data);
+
+  if (error) {
+    return { error: error.message };
+  }
+
+  const { error: spbsError, data: user } = await supabase
+    .from("doctor")
+    .select("*")
+    .eq("email", data.email)
```

Comment:
here there is no email field in doctor's database fix this the login is not working.
<img width="360" alt="Screenshot 2024-07-05 at 08 18 53" src="https://github.com/atlp-rwanda/commanders-medica-backoffice/assets/118519513/cbac6a5a-1f13-457d-9433-56ea2403dbb1">